Tests/NetworkServiceTests/NetworkErrorTests/NetworkError+LocalizedErrorTests.swift

```diff
@@ -0,0 +1,16 @@
+import NetworkService
+import XCTest
+
+final class NetworkErrorLocalizedErrorTests: XCTestCase {
+    func testErrorDescriptionExistsInAPI() {
```

Comment:
Why do we test it?
Isn't that enough to check the `NetworkError` conforms to the `Error`?

Reply:
Probably it's worth to test the value of the localized description for each case instead.

Reply:
You are right, I did just that.